server/lib/transactions.ts

```diff
@@ -176,6 +179,71 @@ export const computeExpenseAmounts = async (
   };
 };
 
+export async function createTransactionsFromPaidStripeExpense(
+  expense: Expense,
+  balanceTransaction: Stripe.BalanceTransaction,
+  charge: Stripe.Charge,
+  { sequelizeTransaction }: { sequelizeTransaction?: SequelizeTransaction } = {},
+) {
+  const paymentMethod = await expense.getPaymentMethod({ transaction: sequelizeTransaction });
+  const payee =
+    expense.fromCollective ||
+    (await Collective.findByPk(expense.FromCollectiveId, { transaction: sequelizeTransaction }));
+
+  // host in this context is the host of the collective getting paid for the expense.
+  // expense.HostCollectiveId -> host of the collective paying the expense.
+  // expense.fromCollectiveId.HostCollectiveId -> host of the payee (receiving the expense money).
+  const host = payee.host || (await Collective.findByPk(payee.HostCollectiveId, { transaction: sequelizeTransaction }));
```

Comment:
That's a good comment. We could further clarify by embedding it in the variable name:

```suggestion
  // host in this context is the host of the collective getting paid for the expense.
  // expense.HostCollectiveId -> host of the collective paying the expense.
  // expense.fromCollectiveId.HostCollectiveId -> host of the payee (receiving the expense money).
  const payeeHost = payee.host || (await Collective.findByPk(payee.HostCollectiveId, { transaction: sequelizeTransaction }));
```